src/i18n/fi.json

```diff
@@ -50,5 +50,6 @@
   "publishingQueue": "Julkaisujono",
   "drafts": "Luonnokset",
   "nickname": "Nimimerkki",
-  "registered": "rekisteröitynyt"
+  "registered": "rekisteröitynyt",
+  "openFullscreenMap": "Avaa karttalla"
```

Comment:
Oh and should be "kartalla", double 'tt' is subject to consonant gradation when inflecting =)